hls-plugin-api/src/Ide/Plugin/Config.hs

```diff
@@ -50,6 +50,7 @@ data Config =
     { checkParents       :: CheckParents
     , checkProject       :: !Bool
     , formattingProvider :: !T.Text
+    , cabalFormattingProvider :: !T.Text
```

Comment:
Hmmm. Arguably with the way plugin config works now, it might make more sense for the user to just enable the one plugin that provides the formatting that they want. But I guess this is maybe better UX. Unclear :thinking:

Reply:
I think it is better and shows a clear way forward for more formatters